modules/ztp-preparing-the-ztp-git-repository.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-preparing-the-ztp-git-repository_{context}"]
+= Preparing the ZTP Git repository
+
+Create a Git repository for hosting site configuration data. The zero touch provisioning (ZTP) pipeline requires read access to this repository.
+
+.Procedure
+
+. Create a directory structure with separate paths for the SiteConfig and PolicyGenTemplate custom resources.
+
+. Add `pre-sync.yaml` and `post-sync.yaml` from `resource-hook-example/<policygentemplates>/` to the path for the PolicyGenTemplate custom resources.
+
+. Add `pre-sync.yaml` and `post-sync.yaml` from `resource-hook-example/<siteconfig>/` to the path for the SiteConfig custom resources.
++
+[NOTE]
+====
+If your hub cluster operates in a disconnected environment you must update the `image` for all four pre and post sync hook CRs.
```

Comment:
"If your hub cluster operates in a disconnected environment, you must update the `image` for all four pre and post sync hook CRs."

Reply:
added comma